entries/j/jetstar.com.json

```
@@ -0,0 +1,19 @@
+{
+	"Jetstar": {
+		"domain": "jetstar.com",
+		"contact": {
+			"facebook": "JetstarAustralia",
+			"twitter": "JetstarAirways"
+		},
+		"keywords": [
+			"transport"
+		],
+		"regions": [
+			"au",
+			"us",
+			"nz",
+			"jp",
+			"sg"
```

These are all the regions from the selection dropdown (in alphabetical order):
```suggestion
			"au",
      "cn",
      "hk",
      "id",
      "jp",
      "kr",
      "lk",
      "my",
      "nz",
      "ph",
      "sg",
      "th",
      "tw",
      "us",
      "vn"
```